extensions/spring-scheduled/deployment/src/main/java/io/quarkus/spring/scheduling/deployment/SpringSchedulingProcessor.java

```diff
@@ -0,0 +1,175 @@
+                }
+                processSpringScheduledAnnotation(scheduledBusinessMethods, bean, method, schedules);
+
+            }
+
+        }
+    }
+
+    void processSpringScheduledAnnotation(BuildProducer<ScheduledBusinessMethodItem> scheduledBusinessMethods,
+            BeanInfo bean, MethodInfo method, List<AnnotationInstance> scheduledAnnotations) {
+        List<AnnotationInstance> schedules = new ArrayList<>();
+        if (scheduledAnnotations != null) {
+            for (AnnotationInstance scheduledAnnotation : scheduledAnnotations) {
+                List<AnnotationValue> springAnnotationValues = scheduledAnnotation.values();
+                List<AnnotationValue> confValues = new ArrayList<>();
+                if (!springAnnotationValues.isEmpty()) {
+                    if (annotationsValuesContain(springAnnotationValues, "fixedRate")
+                            || annotationsValuesContain(springAnnotationValues, "fixedRateString")) {
+                        confValues.add(buildEveryParam(springAnnotationValues));
+                        if (annotationsValuesContain(springAnnotationValues, "initialDelay")
+                                || annotationsValuesContain(springAnnotationValues, "initialDelayString")) {
+                            confValues.addAll(buildDelayParams(springAnnotationValues));
+                        }
+
+                    } else if (annotationsValuesContain(springAnnotationValues, "fixedDelay")) {
+                        throw new IllegalStateException(
+                                "Unsupported @Scheduled param fixedDelay");
+                    } else if (annotationsValuesContain(springAnnotationValues, "cron")) {
+                        if (annotationsValuesContain(springAnnotationValues, "initialDelay")) {
+                            throw new IllegalStateException(
+                                    "Invalid @Scheduled method '" + method.name()
+                                            + "': 'initialDelay' not supported for cron triggers");
+                        }
+                        confValues.add(buildCronParam(springAnnotationValues));
+                    }
+
+                }
+                AnnotationInstance regularAnnotationInstance = AnnotationInstance.create(QUARKUS_SCHEDULED,
+                        scheduledAnnotation.target(), confValues);
+                schedules.add(regularAnnotationInstance);
+            }
+            if (schedules != null) {
+                scheduledBusinessMethods.produce(new ScheduledBusinessMethodItem(bean, method, schedules));
+                LOGGER.debugf("Found scheduled business method %s declared on %s", method, bean);
+            }
+        }
+    }
+
+    AnnotationValue buildCronParam(List<AnnotationValue> springAnnotationValues) {
+        String cronExpression = getAnnotationValueByName(springAnnotationValues, "cron").get().asString();
+        cronExpression = cronExpression.replace("${", "{");
```

Comment:
> @aureamunoz, IIUC correctly from your Pull Request, Spring's ability to read the cron expression from `application.properties` using `${}` isn't supported, right? If not, is there a proper error message?
Yes @geoand , it is supported.

Reply:
Is there a test that shows this functionality? I mean adding a cron expression in `application.properties` and making sure that it works.

Reply:
Yes, in `SpringScheduledMethodTest.java` I add the cron expression in `application.properties` and have a test. And `SpringScheduledMethodsBean#checkEverySecondCronConfig` is the method.